solutions/fully-configurable/variables.tf

```diff
@@ -363,7 +365,7 @@ variable "service_credential_secrets" {
         for credential in group.service_credentials : can(regex("^crn:v[0-9]:bluemix(:..*){2}(:.*){3}:(serviceRole|role):..*$", credential.service_credentials_source_service_role_crn))
       ])
     ])
-    error_message = "service_credentials_source_service_role_crn must be a serviceRole CRN. See https://cloud.ibm.com/iam/roles"
+    error_message = "service_credentials_source_service_role_crn must be a serviceRole CRN. [Learn more](https://cloud.ibm.com/iam/roles)."
```

Comment:
`Learn more` -> can be updated for better clarification like Refer this for the allowed/possible roles

Or can we say:
`Provided value of *** is not valid. Refer this for the allowed roles/values`